### PR TITLE
Remove unused variable DXCoilFlag

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
 - Removes unused variable and code
 - Found in only 2 places, one where initialized to false, and another where it's tested.

image

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/AirLoopHVACDOAS.hh [hunk 1]
```diff
@@ -133,7 +133,6 @@ namespace AirLoopHVACDOAS {
 
         bool m_HeatExchangerFlag = false;
         bool SizingOnceFlag = true;
-        bool DXCoilFlag = false;
```
Comment (rraustad) [inline src/EnergyPlus/AirLoopHVACDOAS.hh:136]: 1 unused variable

File: src/EnergyPlus/Autosizing/CoolingCapacitySizing.cc [hunk 2]
```diff
@@ -272,86 +272,81 @@ Real64 CoolingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
                         state.dataRptCoilSelection->coilSelectionReportObj->setCoilLvgAirHumRat(state, this->compName, this->compType, CoilOutHumRat);
                     }
                 } else if (this->curOASysNum > 0 && this->outsideAirSys(this->curOASysNum).AirLoopDOASNum > -1) {
-                    DesVolFlow = this->airloopDOAS[outsideAirSys(this->curOASysNum).AirLoopDOASNum].SizingMassFlow / state.dataEnvrn->StdRhoAir;
-                    if (this->airloopDOAS[outsideAirSys(this->curOASysNum).AirLoopDOASNum].DXCoilFlag) {
-                        this->autoSizedValue = DesVolFlow / 0.00005;
-                    } else {
```
Comment (rraustad) [inline src/EnergyPlus/Autosizing/CoolingCapacitySizing.cc:278]: Variable is used here but never set to true anywhere in code. The reminaing changes are cleanup.

**Comment by Myoldmopar** [comment]
Seems like an extremely minimal change, with a bit of cleanup along the way.  Do you think you could modify the PR title to something more meaningful and we'll get this merged right in.  Thanks @rraustad !

**Comment by Myoldmopar** [comment]
Thanks @rraustad, no issues with this tiny cleanup.  Merging.

**Pull request merged**